Phonebook.Frontend/src/app/shared/components/room-plan/room-plan.component.html

```diff
@@ -1,4 +1,24 @@
-<div class="svg-container" [style.display]="floorPlanSVG ? 'block' : 'none'" #floorplanSVG></div>
+<!-- Container for Smart Spaces -->
+<ng-container *ngIf="floorplan == 'DDR7HausD24' && runtimeEnvironment.smartSpacesUrl">
+  <mat-slide-toggle
```

Comment:
I think we should display it straight away, no asking or toggle required. Just fallback to the old roomplan if there is an error.